README.adoc

```diff
@@ -1,22 +1,20 @@
+**link:https://chronicle.software/services/[Chronicle Microservices Framework]** - Microservices built with Chronicle Services are efficient, easy to build, test, and maintain. Equally importantly they provide exceptional high-throughput, and low latency.
 
-**Chronicle Map** (CM) is a key-value store sharing persisted memory between processes, either on the same server or across networks. CM is designed to store the data off-heap, which means it minimizes the heap usage and garbage collection allowing the data to be stored with sub-microsecond latency. CM is structured key-value store able to support exceptionally high updates and high throughput data e.g. OPRA Market Data with minimum configuration.
+**link:https://chronicle.software/matching-engine/[Chronicle Matching Engine]** - forms the backbone for a resilient and scalable exchange solution. It provides order matching, validation, and risk checks with high capacity and low latency. It has a modular and flexible design which enables it to be used stand-alone, or seamlessly integrated with Chronicle FIX and Chronicle Services.
 
-**Chronicle Queue** (CQ) persists every event and message, recording these in a memory mapped file. Messages can be written concurrently in the same machine or across a network which ensures no messages are ever lost. CQ can serialize, write, read and deserialize realistic business messages in around two microseconds. This ensures our clients have a low latency complete picture of their trading strategy.
+**link:https://chronicle.software/efx/[Chronicle EFX]** - built on Chronicle Microservices, EFX contains components for Aggregation, Pricing, Hedging, Position Keeping, P&L, Market Gateway and Algo containers. EFX allows the customer to use off the shelf functionality built and maintained by Chronicle, or to extend and customise with their own algos and IP - the best compromise of "buy vs build".
 
-"Datagrid" **Chronicle Datagrid (CE)** is a high performance, low latency access gateway, it is used to control manage access to your data and your services across a number of machines. CE can handle millions of events per second with latencies of around one microsecond on a single machine. CE allows you to install and configure your own business rules and can be used in a variety of ways:
+**link:https://chronicle.software/queue/[Chronicle Queue]** and also link:https://chronicle.software/queue-enterprise/[Chronicle Queue Enterprise] - persists every event and message, recording these in a memory mapped file. Messages can be written concurrently in the same machine or across a network which ensures no messages are ever lost. CQ can serialize, write, read and deserialize realistic business messages in around two microseconds. This ensures our clients have a low latency complete picture of their trading strategy.
```

Comment:
> Messages can be written concurrently in the same machine or across a network which ensures no messages are ever lost.
I don't think this sentence is very clear, or it's trying to make too many points?

Reply:
My interpretation of what I think it's trying to say is...

> Queues can be written to and read from concurrently, on the same machine or across the network via replication. (something about persistent messaging means messages delivery is guaranteed even in the case of endpoint downtime?)

Reply:
perhaps
> Using Chronicle Queue for message passing provides an effectively unlimited buffer between producers and consumers and a complete audit trail of every message sent.

Reply:
Yep - agree